3118 central server config through m supply central (#3583)
msupply-foundation/open-msupply: server/README.md

# Server

Open mSupply server is a component of the Open mSupply system:

> - Hosts the server web interface and exposes RESTful and GraphQL APIs for mSupply data.

Reviewer: wondering about this..?
```suggestion
- Hosts the server web interface and exposes RESTful and GraphQL APIs for open mSupply data.
```

and the list would read more naturally with an intro such as:

```
Open mSupply server is a component of the Open mSupply system. The server:
```

Author: fdd9d526e72d661880053dabe425bcfae0c3dbd5 🙏

- Synchronises with central servers which implement `v5` of the mSupply sync API.
- Exposes a dynamic plugin system for customising and extending functionality.

> - Can act as a remote or central omSupply server, see [Synchornisation documentation](https://github.com/msupply-foundation/open-msupply/blob/develop/server/service/src/sync/README.md#glossary)

Reviewer: ```suggestion
- Can act as a remote or central omSupply server, see [Synchronisation documentation](https://github.com/msupply-foundation/open-msupply/blob/develop/server/service/src/sync/README.md#glossary)
```

and perhaps 'omSupply server' should be 'Open mSupply server'?

Explore API available on `http://localhost:8000/graphql` with build in playground or try [online graphiql explorer](https://graphiql-online.com/)

## Open mSupply Central and Remote

Open mSupply server can be launched in both modes, this is controlled via `Site is open mSupply central server` configuration in legacy mSupply site settings, more info in [Sync Docs](https://github.com/msupply-foundation/open-msupply/blob/develop/server/service/src/sync/README.md#open-msupply-central-server) and [Integration Test Docs](https://github.com/msupply-foundation/open-msupply/blob/develop/server/service/src/sync/test/integration/README.md#4-open-msupply-central-server).

> From version 2.0 omSupply would require both legacy and omSupply central server for full configuration/syncrhonisation. To start both omSupply central and omSupply remote:

Reviewer: again - probably fine here, in internal docs: the preference for external documentation is to use 'Open mSupply' in preference to 'omSupply'

Author: Thanks, i'll try to change to use Open mSupply in internal docs too (and in some places I have), it's just so much easier to type omSupply

- Create two sites as per [above "Set up sync with mSupply central" instructions](https://github.com/msupply-foundation/open-msupply/blob/develop/server/service/src/sync/test/integration/README.md#4-open-msupply-central-server)

> - Configure one site to be central server by check `Site is open mSupply central server` entering URL with which remote sites can reach this site in `This site url`

Reviewer: ```suggestion
- Configure one site to be central server by checking `Site is open mSupply central server` and entering the URL with which remote sites can reach this site in the field `This site url`
```
